patches/minecraft/net/minecraft/entity/LivingEntity.java.patch

```diff
@@ -368,14 +368,25 @@
        super.func_70071_h_();
        this.func_184608_ct();
        this.func_205014_p();
-@@ -2232,6 +_,7 @@
+@@ -2231,7 +_,8 @@
+          }
  
           ItemStack itemstack1 = this.func_184582_a(equipmentslottype);
-          if (!ItemStack.func_77989_b(itemstack1, itemstack)) {
+-         if (!ItemStack.func_77989_b(itemstack1, itemstack)) {
++         if (!itemstack1.equals(itemstack, true)) {
```

Comment:
What does this change do

Reply:
Technically this isn't necessary for this PR however it essentially means that an items share tag will be considered when syncing items instead of it's full NBT (items without a share tag will default to full NBT so doesn't break existing behaviour). This allows equipment items with a custom share tag to be considered when syncing.